pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -158,14 +158,17 @@ func (pc *PackageControllerClient) CreateHelmOverrideValuesYaml() (string, []byt
 
 func (pc *PackageControllerClient) generateHelmOverrideValues() ([]byte, error) {
 	var err error
-	endpoint, username, password, caCertContent := "", "", "", ""
+	endpoint, username, password, caCertContent, insecureSkipVerify := "", "", "", "", "false"
 	if pc.registryMirror != nil {
 		endpoint = pc.registryMirror.BaseRegistry
 		username, password, err = config.ReadCredentials()
 		if err != nil {
 			return []byte{}, err
 		}
 		caCertContent = pc.registryMirror.CACertContent
+		if pc.registryMirror.InsecureSkipVerify {
+			insecureSkipVerify = "true"
```

Comment:
You can just pass pc.registryMirror.InsecureSkipVerify down to the helm installation.

Reply:
InsecureSkipVerify is a bool, so at the very least I need to covert it to a string. There could be a more clever way to do that than this ham fisted way, but not sure it would be a huge improvement

Reply:
I looked at that, but this block checks for `if pc.registryMirror != nil {`, so I think this way is better.